syncd/VendorSai.cpp

```diff
@@ -663,7 +663,11 @@ sai_status_t VendorSai::bulkCreate(
 
         default:
             SWSS_LOG_ERROR("not implemented, FIXME");
-            return SAI_STATUS_FAILURE;
+            return SAI_STATUS_NOT_SUPPORTED;
+    }
+
+    if (!ptr) {
```

Comment:
{ to new line, and add error message that obtained api pointer is null

Reply:
also, you are required to SET all statuses vector to failure, since external logic if see api failure, will also want to check statuses